src/darwin/Framework/CHIP/MTRCSRInfo.h

```diff
@@ -54,11 +54,42 @@ API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4))
  */
 @property (nonatomic, copy, readonly) NSData * attestationSignature;
 
+/**
+ * Initialize an MTROperationalCSRInfo by providing all the fields.  This will
+ * ensure that csr and csrNonce match the data in csrElementsTLV.
+ */
 - (instancetype)initWithCSR:(MTRCSRDERBytes)csr
```

Comment:
This should now be `nullable instancetype` I think?

Reply:
We shouldn't change the existing API signature.... I'll just revert the changes to this method and deprecate it.